src/base/projects/Issue.svelte

```diff
@@ -126,10 +125,10 @@
         </div>
         <div
           class="summary-state"
-          class:closed={issue.state !== "open"}
-          class:opened={issue.state == "open"}
+          class:closed={issue.state.status === "closed"}
+          class:opened={issue.state.status === "open"}
```

Comment:
We're using `class:open` bellow, maybe we should do the same here?

Reply:
done ✅